File: CapstoneProject/src/main/java/com/swcguild/capstoneproject/model/AssetType.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.swcguild.capstoneproject.model;

import java.util.List;
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.persistence.OneToMany;
import javax.persistence.Table;

/**
 *
 * @author apprentice
 */
@Entity
@Table(name="asset_types")
public class AssetType {
    @Id
    @GeneratedValue
    @Column(name="asset_type_id")
    private int assetTypeId;
    
    @Column(name="name")
    private String name;
    
    @ManyToOne
    @JoinColumn(name="category_id")//I think this is right
    private String category;
    
    @Column(name="image_path")
    private String imagePath;
    
    @OneToMany(mappedBy="asset")
    private List<Asset> assets;

    public int getAssetTypeId() {
        return assetTypeId;
    }

    public void setAssetTypeId(int assetTypeId) {
        this.assetTypeId = assetTypeId;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getCategory() {
        return category;
    }

    public void setCategory(String category) {
        this.category = category;
    }

    public String getImagePath() {
        return imagePath;
    }

    public void setImagePath(String imagePath) {
        this.imagePath = imagePath;
    }

    public List<Asset> getAssets() {
        return assets;
    }

    public void setAssets(List<Asset> assets) {
        this.assets = assets;
    }
    

    
    
    
    
}

File: CapstoneProject/src/main/webapp/js/assetType.js
/* 
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */


$(document).ready(function(){
   loadAssetTypes();
    
});

function loadAssetTypes() {
    $('#assetType-list').empty();
    $.ajax({
        url: "http://localhost:8080/CapstoneProject/rest/assetTypes"
    }).then(function(data, status) {
       $.each(data, function(index, assetType){
          $('#assetType-list').append('<p>' + assetType.assetId + " " + assetType.category + '</p>');
       });
    });
}